optimum/intel/utils/import_utils.py

```diff
@@ -94,21 +94,43 @@
         _openvino_tokenizers_available = False
 
 if _openvino_tokenizers_available and _openvino_tokenizers_version != "N/A":
-    _compatible_openvino_version = next(
-        (
-            requirement.split("==")[-1]
-            for requirement in importlib_metadata.requires("openvino-tokenizers")
-            if requirement.startswith("openvino==")
-        ),
-        "",
-    )
-    _openvino_tokenizers_available = _compatible_openvino_version == ov_major_version
+    _is_ovt_dev_version = "dev" in _openvino_tokenizers_version
+    _ov_version = importlib_metadata.version("openvino")

```

Comment:
This will break when users install OpenVINO with other methods than `pip install openvino`.

Reply:
Update checks. Add a table for covered scenarios.